config/config.go

```diff
@@ -873,6 +873,11 @@ func (conf *Config) AllNamespacesAccessible() bool {
 	return conf.Deployment.ClusterWideAccess
 }
 
+// IsServerHttps returns true if the server endpoint should use HTTPS. If false, only plaintext HTTP is supported.
+func (conf *Config) IsServerHttps() bool {
```

Comment:
Very minor nit that you can ignore if you like: but note that if you accept this suggestion you'll probably have to update it everywhere else.
```suggestion
func (conf *Config) IsServerHTTPS() bool {
```

Reply:
I originally had it that way, and the the Code editor made the suggestion to change it to camel case. What is our typical code convention here? I was just following the editor blindly :)

Reply:
I think we're inconsistent but it's a generally accepted style for abbreviations in go: https://github.com/golang/go/wiki/CodeReviewComments#initialisms.

Reply:
done. See latest commit